mlmlora.module

```diff
@@ -224,5 +224,5 @@ function mlmlora_form_islandora_entities_citation_form_alter(&$form, &$form_stat
  */
 function mlmlora_islandora_view_object_alter($object, &$content) {
   module_load_include('inc', 'mlmlora', 'includes/utilities');
-  $content += array(mlmlora_get_pdf_stats($object));
+  $content = array(mlmlora_get_pdf_stats($object)) + $content;
```

Comment:
Not sure we care... This might more "elegantly" be handled by using the `#weight` bit in the associative array... Something like:
```php
$content += array(
  'mlml_pdf_stats' => mlmlora_get_pdf_stats($object) + array(
    // Should just need to be lower than the default of 0, to be rendered at the top of the content.
    '#weight' => -10,
  ),
);
```